packages/web-components/fast-components-msft/src/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,135 @@
+import { css } from "@microsoft/fast-element";
+import {
+    disabledCursor,
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import {
+    heightNumber,
+    neutralFillStealthActiveBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+
+export const MenuItemStyles = css`
+    ${display("grid")} :host {
+        outline: none;
+        box-sizing: border-box;
+        height: calc(${heightNumber} * 1px);
+        grid-template-columns: 42px auto 42px;
+        grid-template-rows: auto;
+        justify-items: center;
+        align-items: center;
+        padding: 0;
+        margin: 0 calc(var(--design-unit) * 1px);
+        white-space: nowrap;
+        overflow: hidden;
+        color: ${neutralForegroundRestBehavior.var};
+        fill: ${neutralForegroundRestBehavior.var};
+        cursor: pointer;
+        font-size: var(--type-ramp-base-font-size);
+        line-height: var(--type-ramp-base-line-height);
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--outline-width) * 1px) solid transparent;
+    }
+
+    :host(:${focusVisible}) {
+        border: calc(var(--outline-width) * 1px) solid ${neutralFocusBehavior.var};
+        box-shadow: 0 0 0 calc((var(--focus-outline-width) - var(--outline-width)) * 1px) ${
+            neutralFocusBehavior.var
+        };
+    }
+
+    :host(:hover) {
+        background: ${neutralFillStealthHoverBehavior.var};
+    }
+
+    :host(:active) {
+        background: ${neutralFillStealthActiveBehavior.var};
+    }
+
+    :host(.disabled) {
+        cursor: ${disabledCursor};
+        opacity: var(--disabled-opacity);
+    }
+
+    :host(.disabled:hover) .start,
+    :host(.disabled:hover) .end,
+    :host(.disabled:hover)::slotted(svg) {
+        fill: ${neutralForegroundRestBehavior.var};
+    }
+
+    .content {
+        grid-column-start: 2;
+        justify-self: start;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    .start,
+    .end,
+    ::slotted(svg) {
+        ${
```

Comment:
@radium-v I think I saw you removing the interpolation of strings in css template literals in your minification PR - is this still okay to do?

Reply:
It'll lead to an expression like `${""}` since the transformer handles the string chunks (`.start,.end,::slotted(svg){`) and the regular JS compression removes the comment in the expression.

Unfortunately, I don't think the transformers in place right now are able to remove the comment if it were part of the CSS, since the string chunks can't be processed as real CSS or HTML (we end up with parts like `;\n    color: ` and `"\n    attribute="` which aren't really parsable as anything on their own).

From a technically-focused perspective, the `${ /* comment */ ""}` is the one that collapses to the fewest bytes, but it still can invoke more cycles depending on how many there are if the engine has to stop and fill in all of those gaps at runtime.